src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -649,12 +654,12 @@ type Get(nameChain: string) =
                     Id = id.ToString() }
                   |> GetError.SubModelSelectedItem
                   |> Error
-        |> Result.map (ValueOption.map snd >> ValueOption.toObj >> box)
+        |> Result.map (ValueOption.map snd >> ValueOption.map box >> ValueOption.toObj >> unbox)
```

Comment:
Not part of this PR:
Why am I creating a pair above if I am just going to call
```fsharp
Result.map (ValueOption.map snd)
```

to throw away the first component?  We can simplify this, right?

Reply:
Yes, it appears so. I have a stash that does this, and it appears to work.